modules/react-components/src/components/danger-zone/danger-zone.tsx

```diff
@@ -107,18 +107,18 @@ export const DangerZone: FunctionComponent<DangerZoneProps> = (
             </Header>
             {
                 toggle ?
-                    <Checkbox
+                    (<Checkbox
```

Comment:
Shall we format here?

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/2712/commits/822ffa6c9b9d9f29aad197efcd574ef27c3e7440 & https://github.com/wso2/identity-apps/pull/2712/commits/f04b53ae80f3c688374854052c552a5f698db22f.